data/arabic/articles/c1er5mjnznzo.json

```diff
@@ -0,0 +1,433 @@
+                            "model": {
+                              "text": "تعرض هذه الصورة التجريبية ، حقوق الطبع والنشر لهيئة الإذاعة البريطانية ، خريطة لفرنسا. الصورة في الكتل الثلاثة الأولى وتحتوي على هذا التعليق.\n",
+                              "blocks": [
+                                {
+                                  "type": "fragment",
+                                  "model": {
+                                    "text": "تعرض هذه الصورة التجريبية ، حقوق الطبع والنشر لهيئة الإذاعة البريطانية ، خريطة لفرنسا. الصورة في الكتل الثلاثة الأولى وتحتوي على هذا التعليق.\n",
+                                    "attributes": []
+                                  }
+                                }
+                              ]
+                            }
+                          }
+                        ]
+                      }
+                    }
+                  ]
+                }
+              },
+              {
+                "type": "rawImage",
+                "model": {
+                  "width": 976,
+                  "height": 549,
+                  "locator": "6eab/test/c2696670-e68f-11e9-88d6-81aa2ccc6d4c.jpg",
+                  "originCode": "cpsdevpb",
+                  "copyrightHolder": "BBC"
+                }
+              }
+            ]
+          }
+        },
+        {
+          "type": "text",
+          "model": {
+            "blocks": [
+              {
+                "type": "paragraph",
+                "model": {
+                  "text": "هذا عبارة عن كتلة نصية ، تحتوي على رابط url إلى الصفحة الرئيسية لبي بي سي للطقس.\n\n\n",
+                  "blocks": [
+                    {
+                      "type": "fragment",
+                      "model": {
+                        "text": "هذا عبارة عن كتلة نصية ، تحتوي على رابط url إلى الصفحة ",
+                        "attributes": []
+                      }
+                    },
+                    {
+                      "type": "urlLink",
+                      "model": {
+                        "text": "الرئيسية لبي بي سي للطقس",
+                        "blocks": [
+                          {
+                            "type": "fragment",
+                            "model": {
+                              "text": "الرئيسية لبي بي سي للطقس",
+                              "attributes": []
+                            }
+                          }
+                        ],
+                        "locator": "https://www.bbc.co.uk/weather",
+                        "isExternal": false
+                      }
+                    },
+                    {
+                      "type": "fragment",
+                      "model": {
+                        "text": ".\n\n\n",
+                        "attributes": []
+                      }
+                    }
+                  ]
+                }
+              }
+            ]
+          }
+        },
+        {
+          "type": "video",
```

Comment:
This block is missing the copyright holder which is required

```
Missing required property 'copyrightHolder' for ':article:content:model:blocks:video:model:blocks:aresMedia:model:blocks:image:model:blocks:rawImage:model'
```

Reply:
👍